Api/CmsEntityConverterInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Overdose\CMSContent\Api;
+
+interface CmsEntityConverterInterface
+{
+    const CMS_MEDIA_NODE = 'media';
+    const PAGE_ENTITY_CODE = 'pages';
+    const BLOCK_ENTITY_CODE = 'blocks';
+
+    /**
+     * @return string
+     */
+    public function getCmsEntityType(): string;
+
+    /**
+     * @return string
+     */
+    public function getCmsEntityCode(): string;
+
+    /**
+     * @param array $cmsEntities
+     * @return array
```

Comment:
According to standards between: `@return` and `@param` must be one blank line:
`* @param <>
*
* @return <>`